src/server/utils.ts

```diff
@@ -1,17 +1,15 @@
-import { readFile as fsReadFile, stat as fsStat } from 'fs';
-import { promisify } from 'util';
-
-export const readFile = promisify(fsReadFile);
-export const stat = promisify(fsStat);
-
 /**
  * Utility timer function for Nodejs performance profiling.
```

Comment:
Remove the `Nodejs` part of the comment

Reply:
Probably also state. "Returns a duration for `0` in environments that do not support `performance`."